hera_sim/visibilities/vis_cpu.py

```diff
@@ -316,58 +269,71 @@ def _base_simulate(self, crd_eq, I_sky):
             myid = self.mpi_comm.Get_rank()
             nproc = self.mpi_comm.Get_size()
 
+        if self.correct_source_positions:
+            # TODO: check if this is the right time to be using...
+            ra, dec = self.correct_point_source_pos(
+                data_model, obstime=data_model.uvdata.time_array[0]
+            )
```

Comment:
@steven-murray This will fail because `uvdata.time_array` is a float but `correct_point_source_pos` expect a string or a Quantity.